### PR TITLE
Preparation for release 1.0.9

### PR DESCRIPTION
**Pull request by matsduf**
* Updated Changes
* Bumped version in GUI.pm

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -1,5 +1,16 @@
 Release history for Zonemaster component Zonemaster-GUI 
 
+v1.0.9 2018-01-12
+Natural Language support:
+- Introducing Danish in GUI (#202, #206, #200, #201, #207, #208)
+
+Fixed:
+- Fixed: Fix broken travis (#203, #205)
```
Comment (mattias-p) [inline Changes]: Lots of fixed here. Maybe remove `Fixed: `?

File: Changes [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+- Update (corrected) gui-faq-en.md (#204)
+- Fixed: Broken style URL in the no-JS interface (#187, #188)
+- Fixed: FAQ doesn't work when browsers preffered language is not en/fr/sv (#184, #185)
+- Module::Install need to be included (#209)
```
Comment (mattias-p) [inline Changes]: Maybe reword this to express what was done rather than needed to be done?

**Comment by matsduf** [comment]
@sandoche2k, can you review this?
